src/tree/constants.ts

```diff
@@ -18,126 +18,145 @@ const githubDirComment = `${commentLeader} Markdown files that GitHub looks for,
+  "https://nodejs.org/en/docs/guides/debugging-getting-started/";
+const rundotshLink = "http://run.sh/";
+const sentryLink = "https://sentry.io/welcome/";
+const serverjsLink = "https://serverjs.io/";
+const shrinkwrapLink =
+  "https://docs.npmjs.com/configuring-npm/shrinkwrap-json.html";
+const travisLink = "https://travis-ci.com/";
+const tsConfigLink = "https://www.typescriptlang.org/";
+const tslintLink = "https://palantir.github.io/tslint/";
+const watchmanLink = "https://facebook.github.io/watchman/";
+const webpackLink = "https://webpack.js.org/";
+const yeomanLink = "https://yeoman.io/authoring/storage.html";
 
 export const commonInfoLinks: Record<string, string> = {
-  'ecosystem.config.js': ecosystemLink,
-  'nodemon.json': nodemonLink,
-  'tslint.build.json': tslintLink,
-  'tslint.json': tslintLink,
-  'npm-shrinkwrap.json': shrinkwrapLink,
-  'run.sh': rundotshLink,
-  '.editorconfig': editorconfigLink,
-  'Gemfile.lock': gemfileLink,
-  'npm-debug.log': npmdebugLink,
-  'manifest.json': manifestLink,
-  'materialize.css': materializeLink,
-  '.flowconfig': flowconfig,
-  '.dockerignore': dockerLink,
-  'docker-compose.yml': dockerLink,
-  'Dockerrun.aws.json': dockerrunLink,
-  'buildspec.yml': buildspecLink,
-  'server.js': serverjsLink,
-  'webpack.config.js': webpackLink,
-  '.npmignore': npmLink,
-  '.buckconfig': buckbuildLink,
-  '.env.development': envvarsLink,
-  '.env.production': envvarsLink,
-  'RunWorker.js': npmrunworkerLink,
-  'Scheduler.js': npmschedulerLink,
-  '.npmrc': npmrcLink,
-  '.yo-rc.json': yeomanLink,
-  'remoteDebug.js': remotedebugLink,
-  '.eslintignore': eslintignoreLink,
-  '.gitattributes': gitattributesLink,
-  '.travis.yml': travisLink,
-  '.watchmanconfig': watchmanLink,
-  'babel.config.js': babelLink,
-  '.babelrc': babelLink,
-  'jest.config.js': jestjsLink,
-  'unittest-jest.config.js': jestjsLink,
-  'metro.config.js': metroLink,
-  'gulpfile.js': gulpfileLink,
-  'jsdoc.json': jsdocLink,
-  'jsdoc.conf.json': jsdocLink,
-  'gatsby-browser.js': gatsbyLink,
-  'gatsby-config.js': gatsbyLink,
-  'gatsby-node.js': gatsbyLink,
-  'gatsby-ssr.js': gatsbyLink,
-  'tsconfig.json': tsConfigLink,
-  '.gitignore': gitIgnoreLink,
-  '.eslintrc.js': esLintLink,
-  '.eslintrc': esLintLink,
-  '.env': dotEnvLink,
-  '.huskyrc.js': npmhuskyLink,
-  '.prettierrc': prettierLink,
-  '.prettierrc.json': prettierconfLink,
-  '.prettierignore': prettierignoreLink,
-  'cypress.json': cypressLink,
-  'package-lock.json': packagelockjsonLink,
-  'package.json': packagejsonLink,
-  '.detoxrc.js': detoxLink,
-  '.detoxrc.json': detoxLink,
-  '.detoxrc': detoxLink,
-  'detox.config.js': detoxLink,
-  'detox.config.json': detoxLink,
+  "ecosystem.config.js": ecosystemLink,
+  "nodemon.json": nodemonLink,
+  "tslint.build.json": tslintLink,
+  "tslint.json": tslintLink,
+  "npm-shrinkwrap.json": shrinkwrapLink,
+  "run.sh": rundotshLink,
+  ".editorconfig": editorconfigLink,
+  Gemfile: gemfileLink,
```

Comment:
```suggestion
  "Gemfile": gemfileLink,
```

Reply:
@ralph-dev I couldn't test "Gemfile" since it automatically goes back to Gemfile... just before commiting it let me say that I tried Podfile without "" and it worked well... and @Manpreet-Bhatti tested putting "" to it but he received an error 

![](https://files.slack.com/files-pri/TANN2PNBE-F018XQVMYR4/image.png)

Reply:
Hmm ok that makes no sense.
What about ["Gemfile"] ?

Reply:
> Hmm ok that makes no sense.
> What about ["Gemfile"] ?

no error this way... let me check if it works properly

Reply:
yep! works fine!